apps/ios/metro.config.js

```diff
@@ -5,16 +5,27 @@
  * @format
  */
 
-const {getWatchFolders} = require('@uifabricshared/build-native');
+const { getDefaultConfig } = require('metro-config');
+const { getWatchFolders } = require('@uifabricshared/build-native');
 
-module.exports = {
-  watchFolders: getWatchFolders(),
-  transformer: {
-    getTransformOptions: async () => ({
-      transform: {
-        experimentalImportSupport: false,
-        inlineRequires: false,
-      },
-    }),
-  },
-};
+module.exports = (async () => {
+  const {
+    resolver: { sourceExts, assetExts },
+  } = await getDefaultConfig();
+  return {
+    watchFolders: getWatchFolders(),
+    transformer: {
+      babelTransformerPath: require.resolve('react-native-svg-transformer'),
```

Comment:
What is this?

Reply:
This allows us to do something like `import Logo from './logo.svg'` directly, which IconTest.tsx uses.

Reply:
Any chance it has got to do with the iOS CI hanging?

Reply:
I certainly hope not, since Win32 uses react-native-svg-transformer and has no problems with it at all.

Reply:
At the end of the day, removing the changes to metro.config.js seems to get things to satisfy the CI. This only affects the test app, so I don't think this will be a huge deal for anyone else who wants to consume FURN Icon.